purchases/src/main/java/com/revenuecat/purchases/interfaces/GetSkusResponseListener.kt

```diff
@@ -23,7 +23,7 @@ interface GetSkusResponseListener {
      *
      * @param skus List of [SkuDetails] retrieved after a successful call to fetch [SkuDetails]
      */
-    fun onReceived(skus: List<SkuDetails?>)
+    fun onReceived(skus: List<SkuDetails>)
```

Comment:
would be cool for our API testers to catch this.

Reply:
They did! That's how I noticed

Reply:
oh!! 🥳

Reply:
Yay 🥳